MISSION.md

```diff
@@ -76,17 +76,22 @@ Meeting recordings will be posted to a YouTube channel for offline viewing.
 
 A public Google Group has been created to facilitate offline discussion outside of Github issues or Pull Requests.
 
-https://groups.google.com/forum/#!forum/open-cluster-management open-cluster-management@googlegroups.com
```

Comment:
does this exist?

Reply:
it should have been created. I will double check that.

Reply:
It does -- I have the privacy settings set right now but will open it up once this document solidifies.